lib/Ed25519VerificationKey2020.js

```diff
@@ -60,29 +59,17 @@ export class Ed25519VerificationKey2020 extends LDVerifierKeyPair {
    * @param {object} [options={}] - Options hashmap.
    * @param {Uint8Array|Buffer} [options.seed] - A 32-byte array seed for a
```

Comment:
A Buffer is an instance of Uint8Array.

```suggestion
   * @param {Uint8Array} [options.seed] - A 32-byte array seed for a
```